classes/teacher.rb

```diff
@@ -3,7 +3,7 @@
 class Teacher < Person
   attr_accessor :specialization
 
-  def initialize(specialization, name, age, id:nil)
+  def initialize(specialization, name, age, id: nil)
     super(name, age, id: id)
```

Comment:
- [ ] There should be unit tests for the student classes, Kindly make sure to test all methods of the student class !!